pilot/pkg/model/telemetry.go

```diff
@@ -424,12 +456,19 @@ func (t *Telemetries) telemetryFilters(proxy *Proxy, class networking.ListenerCl
 		}
 		_, logging := tml[k]
 		_, metrics := tmm[k]
+
+		var logsFilter *tpb.AccessLogging_Filter
+		switch p.Provider.(type) {
+		case *meshconfig.MeshConfig_ExtensionProvider_Stackdriver:
```

Comment:
this is not right. the filter should not be only taken for Stackdriver.  why is this change being made?

Reply:
in function `buildHTTPTelemetryFilter`, there's a comment `Only prometheus and SD supported currently`

Reply:
you're right, updated the implements.